### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by stefanpenner**
These trackings provide little value compared to the cost of sanitizing them.

### CODE REVIEW

File: PERF_GUIDE.md [hunk 1]
```diff
@@ -238,10 +238,15 @@ higher.  `npm install -g broccoli-viz@^3.0.3`.
 To generate visualization:
 
 + `BROCCOLI_VIZ=1 ember build`
+<<<<<<< HEAD
 + `broccoli-viz broccoli-viz.0.json > broccoli-viz.0.dot`
 + `dot -Tpng broccoli-viz.0.dot > broccoli-viz.0.png`
+=======
```
Comment (btecu) [inline PERF_GUIDE.md]: Looks like a merge conflict

**Comment by stefanpenner** [comment]
@rwjblue bugfix beta? or..

File: Analytics.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Analytics:
+
+Ember-cli tracks (only) the following data points,:
```
Comment (tomdale) [inline Analytics.md]: Extra comma at end

File: Analytics.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+* ember-cli version
+* build/rebuild/live-reload times
+* why a rebuild occured (add/remove/mutation of a file)
```
Comment (tomdale) [inline Analytics.md]: s/occured/occurred/

File: Analytics.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* how many errors occure
```
Comment (tomdale) [inline Analytics.md]: s/occure/occured/

File: Analytics.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* See if there are upgrade blockers (users stuck on old version)
+* Have a broad idea for build performance (to make sure we don't regress, and how to prioritize)
+* See which high level features are used (serve/test/build/generate etc.)
+* See an increase/decrease in error rates
+
+## Links to each code-point where ember-cli emits tracking information
+
+* command name (ember server, ember exam, etc): to understand what high level features are used
+  * https://github.com/ember-cli/ember-cli/blob/2da9de596370c0e78ea0c0c3ffcd6a551d2863a9/lib/models/command.js#L277
+* file mutation that triggered a build (if it was a change, add or remove)
```
Comment (tomdale) [inline Analytics.md]: Is this metric useful? What is it used for?

File: Analytics.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  * https://github.com/ember-cli/ember-cli/blob/e798849bde44b54171be08d4fb9ac0ecc6cacfe6/lib/models/server-watcher.js#L31
+  * https://github.com/ember-cli/ember-cli/blob/e798849bde44b54171be08d4fb9ac0ecc6cacfe6/lib/models/server-watcher.js#L41
+* build error: the name of the error
+  * https://github.com/ember-cli/ember-cli/blob/6ec50a1fd21d961f0b0e2ca4daf66a8e7dea6417/lib/models/watcher.js#L32-L34
+* build/rebuild time
+  * https://github.com/ember-cli/ember-cli/blob/503ede1fcb5224d54dc36f82af84550a91d90f26/lib/tasks/build.js#L33
+  * https://github.com/ember-cli/ember-cli/blob/503ede1fcb5224d54dc36f82af84550a91d90f26/lib/tasks/build.js#L44
+* live reload
+  * https://github.com/ember-cli/ember-cli/blob/503ede1fcb5224d54dc36f82af84550a91d90f26/lib/tasks/server/livereload-server.js#L167
+  * https://github.com/ember-cli/ember-cli/blob/503ede1fcb5224d54dc36f82af84550a91d90f26/lib/tasks/server/livereload-server.js#L181
```
Comment (tomdale) [inline Analytics.md]: Should also include:

1. How to opt-out
2. Who has access to the information

File: Analytics.md (hunk 5 above)
Comment (stefanpenner) [inline Analytics.md]: I believe we thought it would be useful, it really hasn't been. I'll remove.

File: Analytics.md (hunk 6 above)
Comment (stefanpenner) [inline Analytics.md]: updated

File: Analytics.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+This information is used to:
+
+* See if there are upgrade blockers (users stuck on old version)
+* Have a broad idea for build performance (to make sure we don't regress, and how to prioritize)
+* See which high-level features are used (serve/test/build/generate etc.)
+* See an increase/decrease in error rates
+
+## Still Not comfortable?
+
+We understand! To permanently disable any analytics gathering you can update your project's `.ember-cli` file (or $HOME/.ember-cli for user-wide):
```
Comment (tomdale) [inline Analytics.md]: Second path should be enclosed in backticks also.

**Comment by tomdale** [comment]
Looks good, not sure why tests were failing before

**Comment by stefanpenner** [comment]
@tomdale the tests were passing, but your review requesting changes marked the PR as red.

**Comment by stefanpenner** [comment]
switch to point to release

**Comment by rwjblue** [comment]
Looks like CI is failing

**Comment by tomdale** [comment]
@stefanpenner nah it was definitely Travis and AppVeyor

**Comment by stefanpenner** [comment]
Strange. Wasn't pre squash

**Pull request merged**